File: FindPlacesNearMe/FindPlacesNearMe/Models/Search+CoreDataProperties.m
//
//  Search+CoreDataProperties.m
//  FindPlacesNearMe
//
//  Created by Mobile OS on 4/3/2020.
//  Copyright © 2020 myproject. All rights reserved.
//
//

#import "Search+CoreDataProperties.h"

@implementation Search (CoreDataProperties)

+ (NSFetchRequest<Search *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"Search"];
}

@dynamic searchString;
@dynamic searches;

@end

File: FindPlacesNearMeSwift/FindPlacesNearMeSwift/Model/Search+CoreDataProperties.swift
//
//  Search+CoreDataProperties.swift
//  FindPlacesNearMeSwift
//
//  Copyright © 2020 Amor Rio. All rights reserved.
//

import Foundation
import CoreData


extension Search {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Search> {
        return NSFetchRequest<Search>(entityName: "Search")
    }

    @NSManaged public var searchString: String?
    @NSManaged public var searches: NSSet?

}

// MARK: Generated accessors for searches
extension Search {

    @objc(addSearchesObject:)
    @NSManaged public func addToSearches(_ value: UserLocationSearch)

    @objc(removeSearchesObject:)
    @NSManaged public func removeFromSearches(_ value: UserLocationSearch)

    @objc(addSearches:)
    @NSManaged public func addToSearches(_ values: NSSet)

    @objc(removeSearches:)
    @NSManaged public func removeFromSearches(_ values: NSSet)

}
